modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackService.kt

```diff
@@ -272,11 +275,9 @@ open class PlaybackService : MediaBrowserServiceCompat(), CoroutineScope {
                             LogBuffer.i(LogBuffer.TAG_PLAYBACK, "startForeground state: $state")
                         } catch (e: Exception) {
                             LogBuffer.e(LogBuffer.TAG_PLAYBACK, "attempted startForeground for state: $state, but that threw an exception we caught: $e")
-                            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S &&
-                                e is ForegroundServiceStartNotAllowedException
-                            ) {
+                            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S && e is ForegroundServiceStartNotAllowedException) {
                                 addBatteryWarnings()
-                                SentryHelper.recordException(e)
+                                analyticsTrackerWrapper.track(AnalyticsEvent.PLAYBACK_FOREGROUND_SERVICE_ERROR)
                                 FirebaseAnalyticsTracker.foregroundServiceStartNotAllowedException()
```

Comment:
Should we remove the Firebase event as we now use Tracks?

Reply:
I wasn't sure if we want it or not. I'll remove it.

Reply:
Thanks